TrackletAlgorithm/MatchCalculator.h

```diff
@@ -798,36 +826,52 @@ void MatchCalculator(BXType bx,
     if(newtracklet && goodmatch==true) { // Write out only the best match, based on the seeding 
       switch (projseed) {
       case 0:
+      if((FML1L2 & mask_L1L2) >> projseed) { //projseed 0-7 equivalent to shift
```

Comment:
In this `switch` statement since the case label is the same as the value of `projseed` if you make the amount you shift by the same as the case label it will be more clear to the compiler that these are constant expressions (assuming the shift doesn't go away like @aehart says)

Reply:
also probably the switch statement argument should not be a magic number.